ert_shared/plugins/plugin_manager.py

```diff
@@ -234,6 +234,25 @@ def get_ertscript_workflows(self):
         self.hook.legacy_ertscript_workflow(config=config)
         return config
 
+    def get_forward_model_jobs(self):
```

Comment:
Is this used or to be used?

Reply:
I can remove everything related to this, as it is a different way of loading functions.